src/compressor/zlib/ZlibCompressor.cc

```diff
@@ -161,6 +162,7 @@ int ZlibCompressor::isal_compress(const bufferlist &in, bufferlist &out)
       return -1;
     }
   }
+  compressor_message = -15;
```

Comment:
make a named const for this value?

Reply:
> make a named const for this value?

compressor_message = ZLIB_DEFAULT_WIN_SIZE;

Reply:
yes